mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -169,7 +186,19 @@ def playlistdelete(context, name, songpos):
 
         Deletes ``SONGPOS`` from the playlist ``NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
```

Comment:
Does this match MPD behaviour for deleting an empty playlist? Also a good candidate for a test case.